rcl_action/include/rcl_action/types.h

```diff
@@ -57,7 +57,7 @@ extern "C"
 #define UUID_SIZE 16
 #define uuidcmp(uuid0, uuid1) (0 == memcmp(uuid0, uuid1, UUID_SIZE))
 #define zerouuid (uint8_t[UUID_SIZE]) {0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0}
-#define uuidcmpzero(uuid) uuidcmp(uuid, zerouuid)
+#define uuidcmpzero(uuid) uuidcmp(uuid, (zerouuid))
```

Comment:
Is there any particular reason to make this (and `uuidcmp` above) macros?  It seems like they can just as easily be done as functions, and then you'd get type-checking and no weird pre-processor things either.